docs/sdks/uid2-sdk-ref-java.md

```diff
@@ -63,56 +58,165 @@ The binary is published on the Maven repository:
 
 - [https://central.sonatype.com/artifact/com.uid2/uid2-client](https://central.sonatype.com/artifact/com.uid2/uid2-client)
 
-## Usage for DSPs
+## Initialization
+
+The initialization step depends on the role, as shown in the following table.
+
+| Role        | Create Instance of Class | Link to Instructions                              |
+|:------------| :--- |:--------------------------------------------------|
+| `DSP`       | BidstreamClient | [Usage for DSPs](#usage-for-dsps)                 |
+| `Sharer`    | SharingClient | [Usage for UID2 Sharers](#usage-for-uid2-sharers) |
+| `Publisher` | Uid2PublisherClient | [Usage for Publishers](#usage-for-publishers)     |
```

Comment:
Uid2PublisherClient => `PublisherUid2Client`

Reply:
My bad, good catch! Thank you.